### PR TITLE
shortID for tables

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/app/shared/table/table.component.ts [hunk 1]
```diff
@@ -137,6 +138,22 @@ export class TableComponent implements OnInit, OnDestroy, AfterViewInit {
     return '';
   }
 
+  getAdvancedLink(index: number, tableColumn: TableColumn): string {
+    let linkTmp = tableColumn.linkPattern || '';
```
Comment (adriandumavitagroup) [inline src/app/shared/table/table.component.ts]: please create a method getAdvancedData that will be used for the advancedDisplay and also for the advancedLink. Pass another parameter "type", so you would know what pattern and params you should use.

getAdvancedData(index, tableColumn, type) ..... type = "display" or "link"

So we don't repeat ourselfs, cause those 2 methods basicly do the same thing

File: src/app/shared/table/table.component.ts [hunk 2]
```diff
@@ -161,6 +178,9 @@ export class TableComponent implements OnInit, OnDestroy, AfterViewInit {
     if (tableColumn.advancedDisplay) {
       return this.getAdvancedDisplay(index, tableColumn);
     }
+    // if (tableColumn.advancedLinkFormat) {
+    //   return this.getAdvancedLink(index, tableColumn);
+    // }
```
Comment (adriandumavitagroup) [inline src/app/shared/table/table.component.ts]: i think we can delete this

Comment (sandorbanvitagroup) [reply src/app/shared/table/table.component.ts]: Yes, done.

File: src/app/shared/table/table.component.ts (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/shared/table/table.component.ts]: Done, thx!

**Review by adriandumavitagroup — APPROVED** [review]

**Review by danielrus — APPROVED** [review]

**Pull request merged**